tests/pyspark_delta/test_pyspark_writer.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import chispa
+import pyspark
+from delta import *
+import shutil
+import os
+
+builder = (
+    pyspark.sql.SparkSession.builder.appName("MyApp")
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+    .config(
+        "spark.sql.catalog.spark_catalog",
+        "org.apache.spark.sql.delta.catalog.DeltaCatalog",
+    )
+)
+
+spark = configure_spark_with_delta_pip(builder).getOrCreate()
+
+def test_reference_table_1_append():
+    os.makedirs("tmp")
+
+    # copy over existing reference table
+    shutil.copytree("out/tables/generated/reference_table_1/delta", "tmp/delta")
+    df = spark.read.format("delta").load("tmp/delta")
```

Comment:
I like the idea of copying a pre-existing table. I am ambivalent about using ones from the reader tests; can be nice, but we might want to create ones that are separate as well.

Reply:
Yea, also ambivalent, but actually creating separate ones might be nice cause then the "expected" and "data to append" files could be co-located in the same directory.